R/install-git.r

```diff
@@ -19,11 +21,16 @@
 #' install_git("git://github.com/hadley/stringr.git", branch = "stringr-0.2")
 #'}
 install_git <- function(url, subdir = NULL, branch = NULL, args = character(0),
-                        ...) {
+                        force = FALSE, ...) {
   if (!missing(args))
     warning("`args` is deprecated", call. = FALSE)
 
   remotes <- lapply(url, git_remote, subdir = subdir, branch = branch)
+
+  if (!isTRUE(force)) {
```

Comment:
unfortunately there is no `isFALSE` function in base R, but I can write one easily enough of course.